### PR TITLE
Add GolangShow

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

**Comment by outcoldman** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -240,3 +240,18 @@ JavaScript, 0% воды
...
+<br><hr><br>
+
+
+#### GolangShow (@GolangShow, #GolangShow): [site](http://golangshow.com), [github](https://github.com/GolangShow), [itunes](https://itunes.apple.com/ru/podcast/podkast-golangshow/id1031101295).
+
+Русскоязычный подкаст о Go.
+
+| Ведущие             | twitter                                     | github                                   |
+| :------------------ | ------------------------------------------- | ---------------------------------------- |
+| Артём Андреенко     | [@miolini](https://twitter.com/miolini)     | [@nesteruk](https://github.com/miolini)  |
```
Comment (outcoldman) [inline README.md]: typo in github link: nesteruk should be miolini

Comment (AlekSi) [inline README.md]: :unamused:

**Comment by AveVlad** [comment]
@AlekSi спасибо!

**Pull request merged**